front-end/src/components/svg-icon/SvgIcon.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+
+const preferencesIcon = (
```

Comment:
you can use https://svgr.now.sh/ to convert svg xml code to react code. this way you have access to props, state, and js